### DISCUSSION TITLE
Apple Silicon Developer Install

### DISCUSSION THREAD

**Discussion by project-delphi** [pyg-team/pytorch_geometric#9117] [Show and tell]
Hello,

A short script for a developer install on Apple Silicon (This worked for me on M1 Pro). 

https://gist.github.com/project-delphi/b3b5cc91386997ff882f0a3f04a4b89a

There's probably a way to clean it up and make it more general - grateful for feedback :=)

**Comment by rusty1s** [comment]
You should now be able to use the wheels for `pyg-lib` instead of manual compilation. Definitely worth a try :)
